test/functional/test-viewer.js

```diff
@@ -201,7 +201,7 @@ describe('Viewer', () => {
         'http://www.example.com');
     expect(viewer.getParam('click')).to.equal('abc');
     expect(windowApi.history.replaceState).to.be.calledWith({}, '',
-        '#ampshare=http%3A%2F%2Fwww.example.com%2F');
+        '#click=abc&ampshare=http%3A%2F%2Fwww.example.com%2F');
```

Comment:
Actually I think we're probably supposed to remove the `click` hash param -- my mistake.

@zhouyx to confirm.